docs/existing-vpc.rst

```diff
@@ -62,10 +62,11 @@ For example, if we have the following infrastructure (notation in terraform)::
 		}
 
 		resource "aws_nat_gateway" "main" {
-		  depends_on    = ["aws_internet_gateway.main"]
-		  allocation_id = "${aws_eip.nat.id}"
-		  subnet_id     = "${aws_subnet.public.1.id}"
-		}
+      count         = "${length(aws_subnet.public)}"
```

Comment:
minor one, perhaps best to keep the indentation as tabs?

Reply:
ah yeah I was confused as to why that was happening - my editor converts all tabs. Will fix this